tests/travis/ut_install.sh

```diff
@@ -14,9 +14,10 @@ go get github.com/stretchr/testify
 go get golang.org/x/tools/cmd/cover
 go get github.com/mattn/goveralls
 go get -u github.com/client9/misspell/cmd/misspell
-sudo service postgresql stop
+sudo service postgresql stop || echo no postgresql need to be stopped
```

Comment:
This would eat the error? not only that there's not pgsql?

Reply:
this is what we need, because postgresql is supposed to be not enable, so actually I didn't install it